addons/mass_mailing/models/mailing.py

```diff
@@ -288,6 +289,12 @@ def _compute_calendar_date(self):
             else:
                 mailing.calendar_date = False
 
+    @api.depends('body_arch', 'body_html')
+    def _compute_is_body_empty(self):
+        for mailing in self:
+            body = mailing.body_arch or mailing.body_html
+            mailing.is_body_empty = (not body or tools.html2plaintext(body).strip() == "")
```

Comment:
We have a ``is_html_empty`` tool available for that purpose.

Reply:
It appears that is_html_empty() does not work well for this use case.

When you have tags with no attributes like <p><br></p> for example, the method returns True,
however when you have attributes inside a tag, (which is our case), the method returns False,
this is due to the regex used inside the method which only takes into account spaces inside these tags.

Reply:
Why do we have that kind of void content ?

Reply:
It is introduced by the editor widget, however, it's not clear why the editor is adding inline styles to the `<p>` tag.

it is always embedding the body with `<p style="margin:0px 0 1rem 0;"><br></p>`.

Reply:
Ok, it is the "new void content" then. Previously if was ``<p><br></p>``, hence the check in is html empty. We could update html empty accordingly, like "containing only p / br and maybe a style attribute". Purpose is to keep something simple but that matches current editor "void content" :) .

Reply:
So we change the `is_html_empty` method in tools?

Reply:
Yes, as editor has changed, we have to adapt this tool method ... its purpose is generally used to detect we don't have any real html code in an html field, so we have to adapt to the new editor "void content" definition :)

Reply:
Okay okay, let's change the editor then :)